packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -60,6 +63,24 @@ class ConfigManager implements IConfigManager {
       );
       // determine the path to fetch remote plugins from
       const pluginsCDNPath = getPluginsCDNPath(state.loadOptions.value.pluginsSDKBaseURL);
+      
+      // Get the consent manager if provided as load option
+    const selectedConsentManager = getUserSelectedConsentManager(
+      state.loadOptions.value.cookieConsentManager,
+    );
+
+    if (selectedConsentManager) {
+      // Get the corresponding plugin name of the selected consent manager from the supported consent managers
+      consentManagerPluginName =
+        ConsentManagersToPluginNameMap[
+          selectedConsentManager as keyof typeof ConsentManagersToPluginNameMap
+        ];
```

Comment:
No

Reply:
That's because you haven't defined the type of the map clearly.
Try this:
```
export const ConsentManagersToPluginNameMap: Record<string, PluginName> = {
  oneTrust: PluginName.OneTrust,
};
```

Reply:
Adressed